llvm/utils/repo/Makefile

```diff
@@ -7,7 +7,7 @@
 #
 
 # The path of the directory in which llvm-project-prepo sources is checked out from git.
-LLVM_REPO = ${HOME}/llvm-project-prepo
+LLVM_REPO = ${HOME}/github/llvm-project-prepo

```

Comment:
Not everyone will have the files checked out here (I know I don’t). Perhaps use ?= ([docs](http://www.gnu.org/software/make/manual/html_node/Setting.html#Setting)) to assign if it wasn’t defined already (e.g. on the command-line)

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/llvm-project-prepo/commit/841f014334761d2b2bf926c7063a4085f53f9cf1.